docs/files-created-full.md

```diff
@@ -0,0 +1,54 @@
+# Common Backup Directory Structure
+ 
+## 📁 Common Backup Directory Structure
+
+Assuming your backup target is `/backups/full_2025_07_21/`, the following files may appear:
+
+### 🔹 Root of the Backup Directory
+
+| File Name                      | Purpose                                                                 |
+|-------------------------------|-------------------------------------------------------------------------|
+| `xtrabackup_info`             | Metadata about the backup (start time, MySQL version, etc.)             |
+| `xtrabackup_binlog_info`      | Binlog coordinates at the time of backup                                |
+| `xtrabackup_checkpoints`      | Checkpoint information for recovery                                     |
+| `xtrabackup_logfile`          | Transaction log file copied during backup                               |
+| `xtrabackup_backup_timestamp` | Timestamp of backup completion (optional)                               |
+| `xtrabackup_gtid_info`        | GTID set at backup time (if GTIDs are enabled)                          |
+| `xtrabackup_replica_info`     | Replication metadata (if server is a replica)                           |
+| `backup-my.cnf`               | Snapshot of server configuration at backup time                         |
+| `ib_buffer_pool`              | Optional buffer pool dump (if enabled via `innodb_buffer_pool_dump_at_shutdown`) |
+
+Some files are conditional:
+
+* `xtrabackup_gtid_info` appears only if GTIDs are enabled.
+
+* `xtrabackup_replica_info` replaces `xtrabackup_slave_info` in MySQL 8.4+ terminology.
+
+* `ib_buffer_pool` appears only if buffer pool dump is configured. The `ib_buffer_pool` file is included in the backup set, which makes the backup slightly larger, but the file does not play any role in the restore operation.
+
+### 🔹 Database Subdirectories
+
+Each database (e.g., `mysql/`, `mydb/`, `anotherdb/`) contains:
+
+- `.ibd` files for InnoDB tables (if `innodb_file_per_table=ON`)
+- `.frm` files for non-InnoDB tables or older MySQL versions
+- `.cfg` files for encrypted tables (if encryption is enabled)
```

Comment:
.cfg and .cfp files are never created during backup. only if user does prepare with --export